### PR TITLE
[Stats Refresh] Add Insights > Follower Totals

### LINKED ISSUE #10513: [Stats Refresh] Add Insights > Follower Totals
Add Insights > Follower Totals card.
- [ ] Broken down per platform
- [ ] Hide empty rows
- [ ] no data

Design:

follower_totals

(8)

### PR DESCRIPTION
**Pull request by ScoutHarris**
Fixes #10513 

This adds the `Follower Totals` card to the Insights table.

NOTE: We don't currently have a way to obtain the Social total. So for now, a fake value is displayed.

Hack tip: To show/hide any of the three follower types, in `StatsInsightsStore`, add a `return ""` to any of these three methods:
- `getTotalDotComFollowers`
- `getTotalEmailFollowers`
- `getTotalPublicizeFollowers`

To test:

---
- On a site with WP, Email, and Social followers, go to Stats > Insights.
  - Tip: en.blog.wordpress.com has lots.
- Verify all three rows appear:

followers_all

---
- On a site that has some of the three follower types, go to Stats > Insights.
- Verify only those that have values appear:

followers_some

---
- On a site that has no followers, go to Stats > Insights.
- Verify `No data yet` appears:

followers_none

### CODE REVIEW

**Comment by ScoutHarris** [comment]
@SylvesterWilmott  - 

full_view

**Comment by ScoutHarris** [comment]
https://github.com/wordpress-mobile/WordPress-iOS/issues/10562 created for the Social total.

**Comment by SylvesterWilmott** [comment]
Looks fab!

Will we do the number formatting in one swoop at the end btw?

File: WordPress/Classes/Stores/StatsInsightsStore.swift [hunk 1]
```diff
@@ -162,7 +210,26 @@ extension StatsInsightsStore {
...
+    }
+
+    func getTotalEmailFollowers() -> String? {
+        return state.totalEmailFollowers == "0" ? "" : state.totalEmailFollowers
+    }
+
+    func getTotalPublicizeFollowers() -> String? {
+        // TODO: When the API is able to, return total of all state.publicizeItems formatted/localized.
+        // For now, we'll just show a bogus number.
+        return "666,6666,666"
```
Comment (jklausa) [inline WordPress/Classes/Stores/StatsInsightsStore.swift]: 😈

File: WordPress/Classes/ViewRelated/Stats/Insights/SimpleTotalsCell.xib [hunk 2]
```diff
@@ -66,9 +66,15 @@
                     <constraint firstAttribute="trailing" secondItem="Tyj-5B-MLc" secondAttribute="trailing" constant="16" id="XOL-9m-28D"/>
                     <constraint firstAttribute="trailing" secondItem="cBd-Ut-Uch" secondAttribute="trailing" id="bwo-8W-tHL"/>
                     <constraint firstAttribute="bottom" secondItem="9px-XX-eCP" secondAttribute="bottom" id="e26-py-r2V"/>
+                    <constraint firstItem="w2N-OJ-hIR" firstAttribute="top" secondItem="Tyj-5B-MLc" secondAttribute="bottom" constant="7" id="kke-sY-9lr"/>
                     <constraint firstItem="w2N-OJ-hIR" firstAttribute="top" secondItem="H2p-sc-9uM" secondAttribute="top" id="olb-Fp-tx9"/>
                     <constraint firstItem="9px-XX-eCP" firstAttribute="leading" secondItem="H2p-sc-9uM" secondAttribute="leading" id="qfo-Hn-1HP"/>
                 </constraints>
+                <variation key="default">
+                    <mask key="constraints">
+                        <exclude reference="kke-sY-9lr"/>
```
[the remark this answers is not in the document]
Comment (ScoutHarris) [reply WordPress/Classes/ViewRelated/Stats/Insights/SimpleTotalsCell.xib]: I don't speak storyboard, but basically that's related to this change: https://github.com/wordpress-mobile/WordPress-iOS/pull/10561/commits/7a1f1a34be4b0a49b56dcf6a469698b0b057d306.

In order to get the subtitles to show/hide correctly, I have two top constraints on the rows:
`rowsStackViewTopConstraint` - is installed by default in the storyboard.
`rowsStackViewTopConstraintWithSubtitles` - is _not_ installed by default in the storyboard. (I think this caused the `variation` in the storyboard.)

Then they are set programmatically according to what the cell needs, i.e.:
`rowsStackViewTopConstraint.isActive = !showSubtitles`
`rowsStackViewTopConstraintWithSubtitles.isActive = showSubtitles`

[the remark this answers is not in the document]
Comment (jklausa) [reply WordPress/Classes/ViewRelated/Stats/Insights/SimpleTotalsCell.xib]: ah, cool! I don't think I ever saw non-installed-by-default constraints before.

thanks for the explanation!

**Pull request merged**